frontend/src/shared/components/IconBadgeList/IconBadgeList.tsx

```diff
@@ -0,0 +1,26 @@
+import { PLATFORM_ICON_MAP, TECH_ICON_MAP } from "./constants/iconMap";
+import IconBadge from "./IconBadge/IconBadge";
+import * as S from "./IconBadgeList.styled";
+
+interface IconBadgeListProps {
+  iconBadges: string[];
+}
+
+const iconBadgeMap: Record<string, { label: string; imgUrl: string }> = {
+  ...TECH_ICON_MAP,
+  ...PLATFORM_ICON_MAP,
+};
```

Comment:
```suggestion
const iconBadgeMap: Record<기술 스택 + 플랫폼들의 키 타입, { label: string; imgUrl: string }> = {
  ...TECH_ICON_MAP,
  ...PLATFORM_ICON_MAP,
};
```

여기도 `Record<string, { label: string; imgUrl: string }>` 보다는 조금 더 타입을 좁힐 수 있을 것 같아요!